libs/src/services/discoveryProvider/index.js

```diff
@@ -52,11 +53,11 @@ class DiscoveryProvider {
         })
 
         try {
-          let resp = await Utils.raceRequests(Object.keys(whitelistMap), (url) => {
+          const { response } = await raceRequests(Object.keys(whitelistMap), (url) => {
```

Comment:
is the consumption of ServiceSelection intended to be a separate PR?

Reply:
yep!

Reply:
my thinking was just to get some of the groundwork done here/ moving files around and then actually using this. will send out that PR soon